soh/src/overlays/actors/ovl_Bg_Toki_Swd/z_bg_toki_swd.c

```diff
@@ -100,7 +100,9 @@ void func_808BAF40(BgTokiSwd* this, GlobalContext* globalCtx) {
         globalCtx->csCtx.segment = D_808BBD90;
         gSaveContext.cutsceneTrigger = 1;
     }
-    if (!LINK_IS_ADULT || ((gSaveContext.eventChkInf[5] & 0x20))) {
+
+    if (!LINK_IS_ADULT || (gSaveContext.eventChkInf[5] & 0x20 && !gSaveContext.n64ddFlag) ||
+        CHECK_QUEST_ITEM(QUEST_MEDALLION_FOREST) && gSaveContext.n64ddFlag) {
```

Comment:
i'm thinking this should be
```
... || (CHECK_QUEST_ITEM(QUEST_MEDALLION_FOREST) && gSaveContext.n64ddFlag)
```
i'm thinking it might not behave as expected without the parens, i won't let it stop me from merging it i'm just making a note to myself to change it post-merge